### PR TITLE
Add jobs fields to scheduler statistics

### PR DESCRIPTION
**Pull request by Mykhailenko**

### CODE REVIEW

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java [hunk 1]
```diff
@@ -614,6 +614,12 @@ public void onSuccess(String result) {
                                       json.get("FormattedMeanJobExecutionTime").isString().stringValue());
                             stats.put("TotalJobsCount", json.get("TotalJobsCount").isString().stringValue());
                             stats.put("PendingJobsCount", json.get("PendingJobsCount").isString().stringValue());
+                            stats.put("StalledJobsCount", json.get("StalledJobsCount").isString().stringValue());
+                            stats.put("PausedJobsCount", json.get("PausedJobsCount").isString().stringValue());
+                            stats.put("InErrorJobsCount", json.get("InErrorJobsCount").isString().stringValue());
+                            stats.put("KilledJobsCount", json.get("KilledJobsCount").isString().stringValue());
+                            stats.put("CancelledJobsCount", json.get("CancelledJobsCount").isString().stringValue());
+                            stats.put("FailedJobsCount", json.get("FailedJobsCount").isString().stringValue());
```
Comment (mcompastie) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java]: Minor comment: to improve code clarity by limiting redundancy, you should consider factoring the code in a for structure.

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java [hunk 2]
```diff
@@ -176,6 +188,13 @@ public void schedulerStatsUpdated(HashMap<String, String> stats) {
         r.setAttribute("TotalJobsCount", stats.get("TotalJobsCount"));
         r.setAttribute("PendingJobsCount", stats.get("PendingJobsCount"));
 
+        r.setAttribute("StalledJobsCount", stats.get("StalledJobsCount"));
+        r.setAttribute("PausedJobsCount", stats.get("PausedJobsCount"));
+        r.setAttribute("InErrorJobsCount", stats.get("InErrorJobsCount"));
+        r.setAttribute("KilledJobsCount", stats.get("KilledJobsCount"));
+        r.setAttribute("CancelledJobsCount", stats.get("CancelledJobsCount"));
+        r.setAttribute("FailedJobsCount", stats.get("FailedJobsCount"));
+
```
Comment (mcompastie) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java]: Minor comment: the same remark applies here

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java [hunk 3]
```diff
@@ -91,7 +91,13 @@ public Layout build() {
...
         DetailViewerField pendingJobsCount = new DetailViewerField("PendingJobsCount", "Pending Jobs Count");
         DetailViewerField runningJobsCount = new DetailViewerField("RunningJobsCount", "Running Jobs Count");
+        DetailViewerField stalledJobsCount = new DetailViewerField("StalledJobsCount", "Stalled Jobs Count");
+        DetailViewerField pausedJobsCount = new DetailViewerField("PausedJobsCount", "Paused Jobs Count");
         DetailViewerField finishedJobsCount = new DetailViewerField("FinishedJobsCount", "Finished Jobs Count");
+        DetailViewerField inErrorJobsCount = new DetailViewerField("InErrorJobsCount", "In-Error Jobs Count");
+        DetailViewerField killedJobsCount = new DetailViewerField("KilledJobsCount", "Killed Jobs Count");
+        DetailViewerField cancelledJobsCount = new DetailViewerField("CancelledJobsCount", "Cancelled Jobs Count");
+        DetailViewerField failedJobsCount = new DetailViewerField("FailedJobsCount", "Failed Jobs Count");
         DetailViewerField totalJobsCount = new DetailViewerField("TotalJobsCount", "Total Jobs Count");
```
Comment (mcompastie) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java]: Minor comment: same remark here.

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java (hunk 1 above)
Comment (Mykhailenko) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerController.java]: good point

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java (hunk 2 above)
Comment (Mykhailenko) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java]: good

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java (hunk 3 above)
Comment (Mykhailenko) [reply scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/StatisticsView.java]: good

**Pull request merged**